### ISSUE TITLE
Show total row count option within the config file

### ISSUE THREAD

**Issue by Chardonneaur** [piwik/piwik#20364]
Hi Team,

would it be possible to add as an option within the config file, the possibility to always see the total rows for each table report?

**Comment by sgiehl** [comment]
Yes, should be possible to make the default for showing a totals row configurable in config or UI.

**Comment by Chardonneaur** [comment]
Fixed with: https://github.com/Chardonneaur/AlwaysShowTotalsRow
